.github/workflows/cypress-demo.yml

```diff
@@ -50,19 +45,19 @@ jobs:
       - run: corepack enable
       - run: pnpm install
 
-      - name: Cypress run
+      - name: Install Playwright Browsers
+        run: pnpm exec playwright install --with-deps
```

Comment:
I'm not sure how often we'll be running this workflow and perhaps this change is for another PR, but might be worth leveraging `actions/cache` to speed up this step.

Reply:
Thanks! I'm new to GitHub Actions, so this is helpful.

I'm going to need to re-work the CI setup in the next PR anyway, so I'll look into it then.